modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/StructuresSectionDisplayContext.java

```diff
@@ -33,6 +33,13 @@ public StructuresSectionDisplayContext(
 		super(cmsSiteInitializerConfiguration, httpServletRequest);
 	}
 
+	@Override
+	public String getAPIURL() {
+		return "/o/object-admin/v1.0/object-definitions?filter=" +
+			"objectFolderExternalReferenceCode eq 'L_CMS_CONTENT_STRUCTURES' " +
```

Comment:
Should these be made a constant in CMSSiteInitializerConstants? Is there a chance these values could change in the future in the data engine api?

Reply:
I would also create the constants but... if Bchan sees only one usage it will reject the constants. So, I would create them later, if we use them again.

Reply:
That is a good point, Thanks Marco.